translations/fr.json

```diff
@@ -342,218 +333,209 @@
   "actionbox.city": "La cité de {0}",
   "actionbox.city.level": "niveau {0}",
   "actionbox.village": "Village",
-  "actionbox.unit.frozen": "{0} glacés",
-  "actionbox.unit.kills": "{0}/{1} morts",
+  "actionbox.unit.frozen": "Glacé {0}",
+  "actionbox.unit.kills": "{0}/{1} Morts",
   "actionbox.unit.veteran": "Vétéran",
   "actionbox.unit.new": "Entraîner un nouveau {0} {1}",
-  "actionbox.unit.ability": "Capacité d'unité",
+  "actionbox.unit.ability": "Capacité",
   "actionbox.unit.train": "ENTRAÎNER",
   "actionbox.unit.upgrade": "AMÉLIORER",
-  "actionbox.unit.toomany": "(trop nombreux)",
-  "actionbox.unit.toomany.info": "Cette ville ne peut plus contenir d'unités. Améliorez votre ville pour avoir plus d'espace pour les unités.",
-  "actionbox.building.doit": "LE FAIRE",
-  "actionbox.building.requiredtech": "Vous avez besoin de rechercher {0} pour faire ça, cliquez sur le bouton 'Recherche' .",
-  "actionbox.building.techtree": "RECHERCHES",
-  "actionbox.insufficientfunds": "Vous n'avez pas assez d'étoiles pour l'acheter. Appuyez sur 'Prochain tour' pour avoir plus d'étoiles.",
-  "actionbox.confirm": "Confirmez {0}",
+  "actionbox.unit.toomany": "(Trop nombreux)",
+  "actionbox.unit.toomany.info": "Cette villes ne peut plus contenir d'unités. Améliorer votre ville pour pouvoir en refaire",
+  "actionbox.building.doit": "APPLIQUER",
+  "actionbox.building.requiredtech": "Vous avez besoin de rechercher '{0}' pour faire ça, appuyer sur le boutton 'Arbre Technologique' .",
+  "actionbox.building.techtree": "Arbre Technologique",
+  "actionbox.insufficientfunds": "Vous n'avez pas assez d'étoiles pour acheter ça. Appuyer sur 'Tour suivant' pour avoir plus d'étoiles.",
+  "actionbox.confirm": "Confirmer {0}",
   "actionbox.confirm.info": "Êtes vous sûr de vouloir faire ça ?",
-  "actionbox.confirm.button": "OUI",
-
+  "actionbox.confirm.button": "oui",
```

Comment:
All buttons are in capital, so it's better to keep it. 
For the other changes, i'm not agree. Like "Arbre technoligique". Even if it's closer to the translation, it causes superposition in game because it's too big.